tests/test_logging.py

```diff
@@ -1,25 +1,25 @@
-import os
 import logging
+import os
 import time
 
 import pytest
 from click.testing import CliRunner
 
+from conftest import (
+    KIDS_FIRST_CONFIG,
```

Comment:
A bunch of these will be unused if you choose to use the suggestion I added below. If you don't then I think just KIDS_FIRST_CONFIG is unused 🔥

Reply:
👍